cypress/e2e/expenses/Add.cy.js

```diff
@@ -40,27 +35,15 @@ describe('Add expense - not within current month', () => {
   });
 
   it('checks that the form has been cleared', () => {
-    cy.findByRole('textbox', { name: /expense name/i }).should(
-      'have.value',
-      ''
-    );
+    cy.findByRole('textbox', { name: /expense name/i }).should('have.value', '');
     cy.findByLabelText(/expense date/i).should('have.value', '');
-    cy.findByRole('spinbutton', { name: /expense amount/i }).should(
-      'have.value',
-      ''
-    );
+    cy.findByRole('spinbutton', { name: /expense amount/i }).should('have.value', '');
   });
 
   it('checks that new expense is correctly rendered', () => {
-    cy.get('[data-cy=expenses]')
-      .should('have.length', 1)
-      .and('not.contain', 'You have no prior expenses');
-
+    cy.get('[data-cy=expenses]').should('have.length', 1).and('not.contain', 'You have no prior expenses');
```

Comment:
Do you know if it's still possible to manually wrap chained methods or will prettier force all the methods onto the same line? I personally think line 44 here was more readable before.